app/templates/components/notification-dropdown.hbs

```diff
@@ -0,0 +1,48 @@
+<i class="large mail outline icon">
```

Comment:
Let the icon be the previous size itself. The large icon doesn't go well with the rest of the navbar